### PR TITLE
Rebuild for Python 3.7, GCC 7, R 3.5.1, openBLAS 0.3.2; Update to 0.10.2

### PR DESCRIPTION
**Pull request by xhochy**
Fixes #4, #6, #7, #8, #9

### CODE REVIEW

**Comment by xhochy** [comment]
@conda-forge-admin please rerender

**Comment by xhochy** [comment]
The failing builds here are due to `distutils.ccompiler.new_compiler()` returning  `cc` instead of the new compilers.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,19 +12,24 @@ source:
...
     - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
   host:
     - python
-    - setuptools
+    - pip
     - cffi >=1.8
   run:
     - python
     - cffi >=1.8
```
Comment (jcrist) [inline recipe/meta.yaml]: I'd also remove cffi as a runtime dependency. Most users don't need this (the c extension is used by default), and the ones that do can install it as an optional dependency.

**Comment by jcrist** [comment]
> The failing builds here are due to distutils.ccompiler.new_compiler() returning cc instead of the new compilers.

Is this something we should add a patch in the build for?

**Comment by xhochy** [comment]
> > The failing builds here are due to distutils.ccompiler.new_compiler() returning cc instead of the new compilers.
> 
> Is this something we should add a patch in the build for?

Yes, preparing one now.

**Comment by mariusvniekerk** [comment]
Thanks so much for this on @xhochy

**Pull request merged**